cmd/build-disk.go

```diff
@@ -19,15 +19,15 @@ package cmd
 import (
 	"os/exec"
 
-	"github.com/rancher/elemental-toolkit/pkg/constants"
-	eleError "github.com/rancher/elemental-toolkit/pkg/error"
-	v1 "github.com/rancher/elemental-toolkit/pkg/types/v1"
+	"github.com/rancher/elemental-toolkit/v2/pkg/constants"
+	eleError "github.com/rancher/elemental-toolkit/v2/pkg/error"
+	v2 "github.com/rancher/elemental-toolkit/v2/pkg/types/v2"
```

Comment:
Should we rename `pkg/types/v2` to just `pkg/types`? I don't think it makes sense to version those types more than is done for the entire module.

Reply:
This a good point, yes we could consider having a single package if we do not foresee supporting both at the same time with the same code.